src/manage.py

```diff
@@ -3,8 +3,6 @@
 import sys
 
 if __name__ == "__main__":
-    if not os.path.exists("src/logs"):
```

Comment:
is it safe to remove this?
What about when running in development mode?

Reply:
Should only log into console now or to sentry, we dont have a logger that handles files anymore in the settings :)